tests/app/db.py

```diff
@@ -660,3 +662,44 @@ def create_service_data_retention(
         days_of_retention=days_of_retention
     )
     return data_retention
+
+
+def create_sample_inbound_numbers(service=None):
+    if not service:
+        service = create_service(service_name='sample service 2')
+    service_2 = create_service(service_name='service 2 for inbound')
```

Comment:
i'm not super sure i'm happy about this function randomly creating two arbitrary services - only one of which specifically says inbound. But I appreciate you're just copying the existing behaviour of the `sample_inbound_numbers` fixture. not sure what to do here (if anything)